Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+      "fields": [
+        { "name": "UpSpeed", "type": "float32" },
+        { "name": "DownSpeed", "type": "float32" },
+        { "name": "Height", "type": "float32" },
+        { "name": "StopTime", "type": "float32" },
+        { "name": "DeadPos", "type": "float32" },
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "Size", "type": "ObjBigCatapultSpawner::Size" },
+        { "name": "Mode", "type": "ObjBigCatapultSpawner::Mode" },
+        { "name": "IsDeadTypeCrush", "type": "bool" },
+        { "name": "IsMessageBoot", "type": "bool" }
+      ]
+    },
+    "ObjBilliardSpawner": {
+      "fields": [
+        { "name": "ModelType", "type": "ObjBilliardSpawner::ModelType" },
+        { "name": "NumMaxCollisions", "type": "uint8" }
+      ]
+    },
+    "EventDimChangetoLoopSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "m_PathEaseTime", "type": "float32" },
+        { "name": "TargetDirection", "type": "vector3" },
+        { "name": "m_IsChangeCamera", "type": "bool" },
+        { "name": "m_IsPadCorrect", "type": "bool" },
+        { "name": "m_IsAutoChange2DPath", "type": "bool" },
+        { "name": "m_IsEnableFromFront", "type": "bool" },
+        { "name": "m_IsEnableFromBack", "type": "bool" }
+      ]
+    },
+    "ObjEggmanMedalSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjEggMaskBoxSpawner": {
+      "fields": [
+        { "name": "AttackInterval", "type": "float32" },
+        { "name": "SearchRadius", "type": "float32" },
+        { "name": "DeadTime", "type": "float32" },
+        { "name": "OffsetY", "type": "float32" },
+        { "name": "OffsetX", "type": "float32" }
+      ]
+    },
+    "ObjEggMaskCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" }
+      ]
+    },
+    "ObjMapPartsBaseSpawner": {
+      "fields": [
+        { "name": "IsPlayerTerrain", "type": "bool" },
+        { "name": "ForceGrid", "type": "bool" }
+      ]
+    },
+    "ObjMapPartsBoxSpawner": {
+      "parent": "ObjMapPartsBaseSpawner",
+      "fields": [
+        { "name": "Type", "type": "ObjMapPartsBoxSpawner::Type" },
+        { "name": "Width", "type": "float32" },
+        { "name": "Height", "type": "float32" },
+        { "name": "Length", "type": "float32" }
+      ]
+    },
+    "ObjMapPartsPlaneSpawner": {
+      "parent": "ObjMapPartsBaseSpawner",
+      "fields": [
+        { "name": "Type", "type": "ObjMapPartsPlaneSpawner::Type" },
+        { "name": "Width", "type": "float32" },
+        { "name": "Height", "type": "float32" },
+        { "name": "Length", "type": "float32" }
+      ]
+    },
+    "ObjMapPartsMeshSpawner": {
+      "parent": "ObjMapPartsBaseSpawner",
+      "fields": [
+        { "name": "Type", "type": "ObjMapPartsMeshSpawner::Type" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and parameters with similar functionality found on other objects.